src/client/src/shell/notification/TradeNotification.tsx

```diff
@@ -1,61 +1,56 @@
-import classnames from 'classnames'
+import moment from 'moment'
 import React from 'react'
 import { Direction, Trade, TradeStatus } from 'rt-types'
-import '../../ui/styles/css/index.css'
+import { Bottom, Close, CloseContainer, MetaContainer, MetaTitle, Notification, Status, Top, Traded } from './Styled'
```

Comment:
warning, _nit_, 100% _nit_: 
I think the file pattern we resolve with @bhavesh-desai-scratch would make it lower cased, so `./styled`

Reply:
Because it exports multiple, and not a class as its default.

Reply:
You're right!